test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
+    os::cmd::expect_success "oc cluster up --service-catalog --version=latest"
+    os::test::extended::clusterup::verify_router_and_registry
+    os::test::extended::clusterup::verify_image_streams
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success "oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated"
+    #oc::cmd::expect_success "oc adm policy add-role-to-user view developer -n openshift"
```

Comment:
This was discussed earlier ... to recap
1) when running the extended tests, and doing these steps manually, I needed this add-role-to-user
2) but when running `make test-extended SUITE=clusterup` from my openshift/origin src dir, I needed to comment it out

Otherwise, in ref to your provision.go comment (which the github browser interface won't let me reply to for some reason), we are in fact able to run with the SAR against gets for template when these add-role-to-users are performed.